### DISCUSSION TITLE
V3 Documentation Icon Aliases

### DISCUSSION THREAD

**Discussion by devhoussam1998** [filamentphp/filament#9578] [Help]
https://filamentphp.com/docs/3.x/support/icons#ui-components-icon-aliases

**pagination.previous-button - Button to go to the previous page (right-to-left direction)**

missing `.rtl`

**missing icon aliases**

```
'actions::action-group'
'actions::create-action.grouped'
'actions::edit-action.grouped'
'actions::view-action.grouped'
```

**Comment by leandrocfe** [comment]
Is it a question?

**Reply by devhoussam1998** [reply]
no suggestion
